lib/middleware/anonymous_cache.rb

```diff
@@ -163,6 +163,7 @@ def has_auth_cookie?
       def no_cache_bypass
         request = Rack::Request.new(@env)
         request.cookies['_bypass_cache'].nil? &&
+          !request.path.ends_with?('srv/status') &&
```

Comment:
A full path comparison instead of `ends_with?` might be better here, to prevent using it as a circumvention method, e.g. pinging `/search?q=srv/status` or something like that?

Reply:
Good call! I've made the changes.